sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -3386,6 +3756,40 @@ def hpat_pandas_series_median_impl(self, axis=None, skipna=True, level=None, num
 @sdc_overload_method(SeriesType, 'argsort')
 def hpat_pandas_series_argsort(self, axis=0, kind='quicksort', order=None):
     """
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
+
+    Pandas API: pandas.Series.argsort
+
+    Examples
+    --------
+    .. literalinclude:: ../../../examples/series/series_argsort.py
+       :language: python
+       :lines: 27-
+       :caption: Override ndarray.argsort.
+       :name: ex_series_argsort
+
+    .. code-block:: console
+
+        > python ./series/series_argsort.py
+        0    1
+        1    2
+        2   -1
+        3    0
+        4    3
+        dtype: int64
+
+    .. note::
+
+        Parameters axis, kind, order are currently unsupported by Intel Scalable Dataframe Compiler
+
+    .. seealso::
+
+        `numpy.absolute <https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.argsort.html#numpy.ndarray.argsort>`_
+            Return indices of the minimum values along the given axis.
```

Comment:
Change to: https://docs.scipy.org/doc/numpy/reference/generated/numpy.ndarray.argsort.html
(without underlining)